test/helpers/helpers.go

```diff
@@ -101,6 +102,27 @@ func WaitForExpectedPhaseAndMessage(client test.FrameworkClient, cscName string,
 	})
 }
 
+// WaitForOpSrcExpectedPhaseAndMessage checks if a OperatorSource with the given name exists in the namespace
```

Comment:
nit: `an OperatorSource`

Reply:
I have a follow up PR where I am adding e2e tests. I will fix this in that PR. Too many of the tests have passed ;-)